### ISSUE TITLE
docker-compose up throw a PHP error

### ISSUE THREAD

**Issue by Prakkkmak** [coopcycle/coopcycle-web#441]
First of all i'm french so sorry for bad eng

Hi,
I want to try the application for my new cooperative.
I'm dev student and I don't know many technologies.
I never see docker tech since today.
So I try to follow the little readme:
For the php install the docker throw me an error and I don't find how to resolve this.

```
PS F:\Projets\coopcycle-web> docker-compose up
Building php
Step 1/25 : FROM php:7.1-fpm-alpine
 ---> 11d8eb18681b
Step 2/25 : RUN apk add --no-cache --virtual .persistent-deps         git         icu-libs         zlib
 ---> Using cache
 ---> 91ad67ab03d4
Step 3/25 : ENV APCU_VERSION 5.1.8
 ---> Using cache
 ---> 6cf0a6d18247
Step 4/25 : RUN set -xe     && apk add --no-cache --virtual .persistent-deps         icu-dev         zlib-dev         postgresql-dev     && apk add --no-cache         xvfb         ttf-freefont         fontconfig         dbus         freetype         libpng         libjpeg-turbo         freetype-dev         libpng-dev         libjpeg-turbo-dev     && apk add qt5-qtbase-dev         wkhtmltopdf         --no-cache         --repository http://dl-3.alpinelinux.org/alpine/edge/testing/         --allow-untrusted     && mv /usr/bin/wkhtmltopdf /usr/bin/wkhtmltopdf-origin     && echo $'#!/usr/bin/env sh\nXvfb :0 -screen 0 1024x768x24 -ac +extension GLX +render -noreset & \nDISPLAY=:0.0 wkhtmltopdf-origin $@ \nkillall Xvfb' > /usr/bin/wkhtmltopdf     && chmod +x /usr/bin/wkhtmltopdf     && apk add --no-cache --virtual .build-deps $PHPIZE_DEPS     && docker-php-ext-configure pdo_pgsql --with-pgsql     && docker-php-ext-configure gd         --with-gd         --with-freetype-dir=/usr/include/         --with-png-dir=/usr/include/         --with-jpeg-dir=/usr/include/     && docker-php-ext-install         intl         pdo_pgsql         zip         bcmath         gd     && pecl install         apcu-${APCU_VERSION}     && docker-php-ext-enable --ini-name 20-apcu.ini apcu     && docker-php-ext-enable --ini-name 05-opcache.ini opcache     && apk del .build-deps
 ---> Using cache
 ---> 74c3eec41c45
Step 5/25 : COPY docker/php/php.ini /usr/local/etc/php/php.ini
 ---> Using cache
 ---> 25b70c3a4e73
Step 6/25 : COPY docker/php/install-composer.sh /usr/local/bin/docker-app-install-composer
 ---> Using cache
 ---> 1cc0e8c3261f
Step 7/25 : RUN chmod +x /usr/local/bin/docker-app-install-composer
 ---> Using cache
 ---> e3b2af606b45
Step 8/25 : RUN set -xe     && apk add --no-cache --virtual .fetch-deps         openssl     && docker-app-install-composer     && mv composer.phar /usr/local/bin/composer     && apk del .fetch-deps
 ---> Running in 5c2516e1229b
+ apk add --no-cache --virtual .fetch-deps openssl
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
(1/4) Installing libcrypto1.0 (1.0.2o-r2)
(2/4) Installing libssl1.0 (1.0.2o-r2)
(3/4) Installing openssl (1.0.2o-r2)
(4/4) Installing .fetch-deps (0)
Executing busybox-1.28.4-r1.trigger
Executing ca-certificates-20171114-r3.trigger
OK: 421 MiB in 226 packages
+ docker-app-install-composer
/bin/sh: docker-app-install-composer: not found
ERROR: Service 'php' failed to build: The command '/bin/sh -c set -xe     && apk add --no-cache --virtual .fetch-deps         openssl     && docker-app-install-composer     && mv composer.phar /usr/local/bin/composer     && apk del .fetch-deps' returned a non-zero code: 127
```

Thank's in advance for the help !

**Comment by alexsegura** [comment]
Hello. 

Where are you located? If you are planning to create a co-op, we would really like to get in touch with you to join our federation of local co-ops. 
On the technical side, we provide hosting & support, so that you can concentrate on building your business. But this is not limited to the technical side: we are building a whole network of mutualized services. 

Come build it with us! 
Don't do this alone 😘

Drop us a line on contact@coopcycle.org

--- 

Regarding your error, it says it is not able to execute `docker-app-install-composer`, but it _should_ be able to do it, it is related to the two lines below in `docker/php/Dockerfile`

```
COPY docker/php/install-composer.sh /usr/local/bin/docker-app-install-composer
RUN chmod +x /usr/local/bin/docker-app-install-composer
```

**Comment by alexsegura** [comment]
Oh, sorry, I didn't notice you are on Windows. 

There is an attempt in #388 to make it work on Windows. 
We still need to make sure it doesn't break on MacOSX/Linux. 

I can give a look in a few days.

**Comment by alexsegura** [comment]
Fixed in #508
